test/test_config.py

```diff
@@ -160,60 +144,29 @@ def test_no_scripts_in_top_level_directory(self, control_path):
             "'tools' sub-directory"
         )
 
+    def test_validate_metadata(self, metadata):
+        # Schema URL
+        schema_version = metadata.get("schema_version", DEFAULT_SCHEMA_VERSION)
+        url = f"{BASE_SCHEMA_URL}/{schema_version}.json"
 
-@pytest.mark.highres
-def test_mppncombine_fast_collate_exe(config):
-    pattern = r'/g/data/vk83/apps/mppnccombine-fast/.*/bin/mppnccombine-fast'
-    if 'collate' in config:
-        assert re.match(pattern, config['collate']['exe']), (
-            "Expect collate executable set to mppnccombine-fast"
-            )
-
-
-@pytest.mark.metadata
-def test_validate_metadata(metadata):
-    # Schema URL
-    schema_version = metadata.get("schema_version", DEFAULT_SCHEMA_VERSION)
-    url = f"{BASE_SCHEMA_URL}/{schema_version}.json"
-
-    # Get schema from Github
-    response = requests.get(url)
-    assert response.status_code == 200
-    schema = response.json()
+        # Get schema from Github
+        response = requests.get(url)
+        assert response.status_code == 200
+        schema = response.json()
```

Comment:
Just a thought, but as schema are immutable we could keep a copy of the schema in this repo for convenience.

Reply:
If you thought it was worthwhile.

Reply:
I've added in a default commit for the default metadata schema version (`1-0-0`), so it should be able to retrieve it from github to make the tests a bit more robust.